### PR TITLE
Review AI Test Toolkit Changes

### PR DESCRIPTION
**Pull request by StefanMaron**
In this PR I did address (hopfully) most of the comments I put in the pull request.
Its mostly guideline changes and readability improvements.

### CODE REVIEW

File: src/Tools/AI Test Toolkit/src/API/AITLogEntryAPI.Page.al
Comment (nikolakukrika) [inline src/Tools/AI Test Toolkit/src/API/AITLogEntryAPI.Page.al]: Great addition. Could you add caption please to these fields?

File: src/Tools/AI Test Toolkit/src/Logs/AITLogEntry.Table.al
Comment (nikolakukrika) [inline src/Tools/AI Test Toolkit/src/Logs/AITLogEntry.Table.al]: Lets not put AIT to the Caption. Recommendation - AI Test Log entry.

File: src/Tools/AI Test Toolkit/src/Logs/AITTestMethodLinesCompare.Page.al
Comment (nikolakukrika) [inline src/Tools/AI Test Toolkit/src/Logs/AITTestMethodLinesCompare.Page.al]: Same comment here 
AI Test Method...

File: src/Tools/AI Test Toolkit/src/Logs/AITTestSuiteCompare.Page.al
Comment (nikolakukrika) [inline src/Tools/AI Test Toolkit/src/Logs/AITTestSuiteCompare.Page.al]: AI Test Suite

File: src/Tools/AI Test Toolkit/src/TestSuite/AITTestMethodLine.Table.al
Comment (nikolakukrika) [inline src/Tools/AI Test Toolkit/src/TestSuite/AITTestMethodLine.Table.al]: AI Test Method Line

File: src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuite.Table.al
Comment (nikolakukrika) [inline src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuite.Table.al]: AI Test Suite

File: src/Tools/Test Framework/Test Runner/src/TestMethodLine.Table.al
Comment (nikolakukrika) [inline src/Tools/Test Framework/Test Runner/src/TestMethodLine.Table.al:135]: I think these fields are not editable when surfaced in client, but no harm to be explicit.

**Review by nikolakukrika — APPROVED** [review]
Thanks a lot for doing this change. Could you update the minor comments?
If not we will fix it after it is merged.  Feel free to merge it when you can.

**Comment by StefanMaron** [comment]
@nikolakukrika I can not merge it myself, but I did the changes

**Review by nikolakukrika — APPROVED** [review]

**Review by Aleyenda — APPROVED** [review]

**Comment by nikolakukrika** [comment]
We had conflicts, I have resolved them in a different branch and merged.
Thanks a lot for the review and the fixes!

**Pull request closed without merging**
